src/handler/module.ts

```diff
@@ -71,14 +71,22 @@ export async function handleModule(
           case 'JsonValue':
             replacer(signature.type.pos, signature.type.end, `${nsName}.${typename}`);
             break;
+            
+          case 'JsonValue | null':
+            replacer(signature.type.pos, signature.type.end, `Nullable<${nsName}.${typename}>`);
+            break;
 
           case 'InputJsonValue':
           case 'InputJsonValue | InputJsonValue':
             replacer(
               signature.type.pos,
               signature.type.end,
               `DeepPartial<${nsName}.${typename}>`
-            );
+              );
+              break;
+ 
+          case 'NullableJsonNullValueInput | InputJsonValue':
+            replacer(signature.type.pos, signature.type.end, `Nullable<DeepPartial<${nsName}.${typename}>>`);
```

Comment:
```suggestion
            replacer(signature.type.pos, signature.type.end, `DeepPartial<${nsName}.${typename}> | null`);
```

Or the null should be inside the generics?

```suggestion
            replacer(signature.type.pos, signature.type.end, `DeepPartial<${nsName}.${typename} | null>`);
```